src/gui/plugins/component_inspector/ComponentInspector.cc

```diff
@@ -101,39 +114,57 @@ namespace ignition::gazebo
     /// \brief Nested model or not
     public: bool nestedModel = false;
 
+    /// \brief If a model, keep track of available links
+    public: QStringList modelLinks = {};
+
     /// \brief Whether currently locked on a given entity
     public: bool locked{false};
 
     /// \brief Whether updates are currently paused.
     public: bool paused{false};
 
+    /// \brief Whether simulation is currently paused.
+    public: bool simPaused{true};
+
     /// \brief Transport node for making command requests
     public: transport::Node node;
+
+    /// \brief Transport node for making command requests
+    public: ModelEditor modelEditor;
+
+    /// \brief Air pressure sensor inspector elements
+    public: std::unique_ptr<ignition::gazebo::AirPressure> airPressure;
+
+    /// \brief Altimeter sensor inspector elements
+    public: std::unique_ptr<ignition::gazebo::Altimeter> altimeter;
+
+    /// \brief Imu inspector elements
+    public: std::unique_ptr<ignition::gazebo::Imu> imu;
+
+    /// \brief Joint inspector elements
+    public: std::unique_ptr<ignition::gazebo::JointType> joint;
+
+    /// \brief Lidar inspector elements
+    public: std::unique_ptr<ignition::gazebo::Lidar> lidar;
+
+    /// \brief Magnetometer inspector elements
+    public: std::unique_ptr<ignition::gazebo::Magnetometer> magnetometer;
+
+    /// \brief Pose inspector elements
+    public: std::unique_ptr<ignition::gazebo::Pose3d> pose3d;
+
+    /// \brief Set of callbacks to execute during the Update function.
+    public: std::vector<
+            std::function<void(EntityComponentManager &)>> updateCallbacks;
+
+    /// \brief A map of component type to creation functions.
+    public: std::map<ComponentTypeId, ComponentCreator> componentCreators;
   };
 }
 
 using namespace ignition;
 using namespace gazebo;
 
-//////////////////////////////////////////////////
-template<>
-void ignition::gazebo::setData(QStandardItem *_item, const math::Pose3d &_data)
```

Comment:
With this removed, the `WorldPose` component is now processed as a string. We should make sure that component is also nicely displayed as a pose:

![image](https://user-images.githubusercontent.com/5751272/144679590-c1be6ad1-6f9d-4187-8ba1-051b2ebed0b7.png)